crates/release_plz/src/config.rs

```diff
@@ -144,12 +144,8 @@ pub struct ReleasePrConfig {
 /// Config at the `[[package]]` level.
 #[derive(Serialize, Deserialize, PartialEq, Eq, Debug, Clone)]
 pub struct PackageSpecificConfig {
-    /// Options for the `release-plz update` command (therefore `release-plz release-pr` too).
-    #[serde(flatten)]
-    update: PackageUpdateConfig,
-    /// Options for the `release-plz release` command.
     #[serde(flatten)]
-    release: PackageReleaseConfig,
+    package_config: PackageConfig,
```

Comment:
I'm sorry to ask you for changes, but I don't like this refactoring (in the future I would like to make the config struct more flatten, instead of nested.
Can you revert this change and put the option in this struct (together with `changelog_path` and `changelog_include` which are also common to the `update` and `release` commands)?

Reply:
Any particular reason why you did this change?

Reply:
> Any particular reason why you did this change?

The struct `PackageConfig` had the same definition as the struct `PackageSpecificConfig` (minus the two `changelog_*`)so as I was adding the configuration it seemed to made sense to centralize them.
Though now that you mention that the `changelog_*` configs were also common I am not seeing what the `PackageSpecificConfig` does versus the `PackageConfig`.
My understanding is that `PackageConfig` is the default configuration for packages and the `PackageSpecificConfig` is for the overrides.
Since the `changelog_*` were only in the package specific config I assumed that those could not be set as the default (Workspace) level, but if they should behave the same, maybe we could have a single (flatter) struct?

For the common object I did so as it had been done the same way on the `Workspace` struct so I thought I was following the existing patterns 🤣

PS: No problem if I do end up reverting things, happy to see you are challenging changes!

Reply:
> My understanding is that PackageConfig is the default configuration for packages and the PackageSpecificConfig is for the overrides.

This is also my understanding (I wrote the code a while back!).

> Since the changelog_* were only in the package specific config I assumed that those could not be set as the default (Workspace) level

You should be able to set changelog config at the workspace level. See docs here:
https://github.com/MarcoIeni/release-plz/blob/986bbc4c187258eb644f6b6361e3d25168975550/website/docs/config.md#L50

> maybe we could have a single (flatter) struct?

That's my goal, yes. I want to do a refactoring and remove many of those nested structs. But let's do it in another PR. 

> happy to see you are challenging changes!

Same for me! :)

Reply:
Sounds good, I will make the change to match what `changelog_config` does and do another PR for the structs!
PS: Finally getting back to working on this ;)

Reply:
I'm happy to hear you're back and feeling better 🎉